community/portworx/values-metadata.yaml

```diff
@@ -139,7 +139,7 @@ stork:
   __metadata:
     name: "stork"
```

Comment:
`Values.stork` no longer used?  If yes, just remove this section from the metadata file.